tests/unit/Service/AutoProvisioningServiceTest.php

```diff
@@ -163,6 +163,8 @@ public function providesProvisioningData(): array {
 			[true, false, true, false, false, ['mode' => 'userid', 'auto-provision' => ['enabled' => true, 'display-name-claim' => 'name']], (object)['email' => 'alice@example.net', 'name' => 'Alice']],
 			[true, false, false, true, false, ['mode' => 'userid', 'auto-provision' => ['enabled' => true, 'picture-claim' => 'picture']], (object)['email' => 'alice@example.net', 'picture' => 'http://']],
 			[true, false, false, false, true, ['mode' => 'userid', 'auto-provision' => ['enabled' => true, 'groups' => ['oidc-group']]], (object)['email' => 'alice@example.net', 'picture' => 'http://']],
+			[true, false, false, false, false, ['auto-provision' => ['enabled' => true, 'provisioning-claim' => 'foo', 'provisioning-attribute' => 'bar']], (object)['email' => 'alice@example.net', 'foo' => ['bar']]],
+			[false, false, false, false, false, ['auto-provision' => ['enabled' => true, 'provisioning-claim' => 'foo']], (object)['email' => 'alice@example.net', 'foo' => ['bar']]],
 		];
```

Comment:
Why is 'bar' in an array? You only get simple key value pairs in user info.

Reply:
This is the only way I found to make the code go past the login exception. Am I missing something?
Also #149 says:
```
\"eduperson_entitlement\":[ \"blabla\" ],
\"eduperson_principal_name\":\"john.doe\", // username
```
So makes sense?

Reply:
I see nothing in https://openid.net/specs/openid-connect-core-1_0.html#Claims that prohibits using arrays. Yes the standard claims are all simple kv-pairs. But we have custom claims here. The only constraint is:
> If the UserInfo Response is signed and/or encrypted, then the Claims are returned in a JWT and the content-type MUST be application/jwt

Also nothing nothing in https://tools.ietf.org/html/rfc7519 (JWT) which prohibits arrrays.
> Claim Value 
> The value portion of a claim representation.  A Claim Value can be any JSON value.

Reply:
THX - one never stops learning new things.